packages/manager/apps/web/client/app/hosting/cdn/shared/leave-confirm-settings/leave/hosting-shared-leave-settings.html

```diff
@@ -1,91 +1,80 @@
 <oui-modal
     data-heading="{{:: 'hosting_cdn_shared_modal_leave_title' | translate}}"
-    data-primary-action="$ctrl.primaryAction()"
+    data-primary-action="$ctrl.leave()"
     data-primary-label="{{'hosting_cdn_shared_modal_leave_btn_validate' | translate}}"
-    data-secondary-action="$ctrl.secondaryAction()"
+    data-secondary-action="$ctrl.goBack()"
     data-secondary-label="{{:: 'hosting_cdn_shared_modal_leave_btn_cancel' | translate}}"
     type="warning"
-    data-on-dismiss="$ctrl.modal.close(true)"
+    data-on-dismiss="$ctrl.goBack()"
 >
     <!--Description-->
     <div
         class="oui-paragraph"
         data-ng-bind=":: 'hosting_cdn_shared_modal_leave_info' | translate"
     ></div>
 
-    <!--Always online-->
-    <div class="mb-2">
-        <div class="row">
-            <div class="text-right col-md-6">
-                <p class="oui-heading_5 mb-1"><strong>Always online</strong></p>
-            </div>
-            <div class="text-left col-md-6">
-                <p
-                    class="mb-1"
-                    data-ng-bind="::'hosting_cdn_shared_state_' + ($ctrl.model.alwaysOnline.enabled ? 'enable' : 'disabled') | translate"
-                ></p>
-            </div>
+    <dl class="oui-description oui-description_horizontal">
+        <dt><strong>Always online</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.alwaysOnline.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>HTTP/2</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.http.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Devmode</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.devmode.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Brotli</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.brotli.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Cache rule</strong></dt>
+        <dd
+            data-ng-repeat="rule in ::$ctrl.model.rules track by $index"
+            data-ng-bind="::rule.name"
+        ></dd>
+        <div data-ng-if="$ctrl.model.cors">
+            <dt><strong>CORS</strong></dt>
```

Comment:
do you think it should come from a constant file?

Reply:
Same than other props